### ISSUE TITLE
New 6.6.2 is duplicate of 2.4.x

### ISSUE THREAD

**Issue by tghosth** [OWASP/ASVS#2238]
As pointed out by @jmanico, 6.6.2 is duplicate of 2.4.x. Need to consider how we handle this.

| # | Description | L1 | L2 | L3 | CWE |
| :---: | :--- | :---: | :---: | :---: | :---: |
| **6.6.2** | [ADDED] Verify that slow hashing functions are used for password storage, with appropriate parameter settings as outlined below. | | ✓ | ✓ | 916 |

**Comment by jmanico** [comment]
I'm ok to move 2.4.x to v6 if you want. It can go in either place, honestly.

**Comment by tghosth** [comment]
@elarlang @danielcuthbert @cronchie @ImanSharaf @ryarmst @TobiasAhnoff

**Comment by ImanSharaf** [comment]
I’m okay with this approach, but let’s add Argon2 as a recommended hashing function for enhanced security.

**Comment by jmanico** [comment]
This is done already, Argon2 was added to the 2.4 text above the requirements

**Comment by tghosth** [comment]
Yeah I think @danielcuthbert still needs to decide what to do about this duplicatoin

**Comment by elarlang** [comment]
>  ".. as outlined below."

If not removed as duplicate, then it requires a modification - requirement must be understandable as stand-alone piece of text.

**Comment by danielcuthbert** [comment]
I think moving it is valid too, so actioning that now alongside making it more understandable.

**Comment by danielcuthbert** [comment]
Created a new PR https://github.com/OWASP/ASVS/pull/2276

not sure what we do about the text in 2.4.x, that Ill take advice from @jmanico @tghosth @elarlang on

**Comment by tghosth** [comment]
@danielcuthbert how do you think we handle 2.4.6 as that is the only remaining requirement in that section. Do you think it makes sense to move it to v6 as well?


| # | Description | L1 | L2 | L3 | CWE | [NIST &sect;](https://pages.nist.gov/800-63-3/sp800-63b.html) |
| :---: | :--- | :---: | :---: | :---: | :---: | :---: |
| **2.4.6** | [ADDED, SPLIT FROM 2.1.2] Verify that the application is protected against a denial of service attack caused by processing an overly long password. | | ✓ | ✓ | | |

**Comment by danielcuthbert** [comment]
No, i feel this is much better suited for v2.1

**Comment by tghosth** [comment]
Ok, I added comments in the PR
